src/views/projectExplorer/library.ts

```diff
@@ -76,6 +77,12 @@ export default class Library extends ProjectExplorerTreeItem {
     if (![LibraryType.systemLibrary, LibraryType.defaultUserLibrary].includes(libraryType) && !variable) {
       this.contextValue += ContextValue.configurable;
     }
+
+    this.contextValue += 
+      (position === 'first' ? ContextValue.first : '') +
+      (position === 'last' ? ContextValue.last : '') +
+      (position === 'middle' ? ContextValue.middle : ''); +
```

Comment:
There is an extra semicolon here that should be removed or else the last check won't happen.